### PR TITLE
fix(publish) make the published items move to the published stage

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/features/content_publish.feature [hunk 1]
```diff
@@ -35,7 +35,8 @@ Feature: Content Publishing
       """
       And we publish "#archive._id#"
       Then we get OK response
-      And we get existing resource
+      When we get "/archive/#archive._id#"
```
Comment (amagdas) [inline server/features/content_publish.feature]: If this is needed, it means the returned doc is not having all the updates and the client side needs to refetch the item.

Comment (marwoodandrew) [inline server/features/content_publish.feature]: True, will fix

**Comment by amagdas** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Pull request merged**